.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## Description of changes
+
+<!-- What have you implemented in this PR? -->
+
+## Checklist
+
+<!-- [x] if you have completed the task -->
+<!-- [n/a] if the task is not relevant -->
+
+- [ ] Content has no typos
+- [ ] Correct inline validation errors
+- [ ] Back link and next page navigation
```

Comment:
Doesn't really make sense to have the front end checks in the application repo 😃

Reply:
I have removed them